symfony/framework-bundle/CVE-2022-xxx.yaml

```diff
@@ -0,0 +1,14 @@
+title:     "CVE-2022-xxxx: CSRF token missing in forms"
+link:      https://symfony.com/cve-2022-xxxx
+cve:       CVE-2022-xxxx
+branches:
+    5.3.x:
+        time:     2022-01-29 12:00:00
+        versions: ['5.3.14', '5.3.14']
```

Comment:
maybe with a sign like `=` ?
Or `>=5.3.24, <=5.3.24`

Reply:
why? this would be exactly the same isn't it?

Reply:
see CI check